schemas/account.js

```diff
@@ -1,6 +1,6 @@
 const {schemas} = require('bedrock-validation');
 
-const postAccounts = {
+const byEmail = {
```

Comment:
this name is suspicious.  Name appropriate to what the validator is used for.

Reply:
renamed to create.